Recrutify/Recrutify.Host/Configuration/Profiles/ProjectProfile.cs

```diff
@@ -17,7 +17,12 @@ public ProjectProfile()
                 .ForMember(dest => dest.Mentors, conf => conf.MapFrom(src => src.Mentors.Select(m => new Staff() { UserId = m })))
                 .ForMember(dest => dest.Interviewers, conf => conf.MapFrom(src => src.Interviewers.Select(i => new Staff() { UserId = i })))
                 .ForMember(dest => dest.Recruiters, conf => conf.MapFrom(src => src.Recruiters.Select(r => new Staff() { UserId = r })));
-            CreateMap<Project, ProjectDTO>().ReverseMap();
+            CreateMap<ProjectDTO, Project>().ReverseMap();
```

Comment:
you don't need ProjectDTO -> Project map, only reversed

Reply:
done